api-tests/platform/targets/tgt_dev_apis_stdc/target.cmake

```diff
@@ -15,6 +15,53 @@
 # * limitations under the License.
 #**/
 
+###############################################################################
+# FUNCTION: _create_psa_storage_exe()
+#  Function for generating PSA Storage test binaries. This function requires
+#  PSA_STORAGE_LIB_FILENAME to be specificed on the cmake command line,
+#  where the symbol is defined as the full path to the external PSA storage
+#  library to test. e.g.
+#    cmake ... -DPSA_STORAGE_LIB_FILENAME=/wdir/usr/lib/libpsastorage.so
+# ARGUMENTS:
+#   _exe_name     Name of the test binary to generate
+#   _api_dir      PSA storage API directory name e.g. internal_trusted_storage
+#                 or protected_storage
+###############################################################################
+function(_create_psa_storage_exe _exe_name _api_dir)
+
+	if(NOT DEFINED PSA_STORAGE_LIB_FILENAME)
+		message(FATAL_ERROR "ERROR: PSA_STORAGE_LIB_FILENAME undefined.")
+	endif()
+
+	# Create the PSA PS Storage test binary.
+	set(EXE_NAME ${_exe_name})
+
+	# Define PSA_LIB_NAME to be the name of the PSA Storage library to be tested.
+	get_filename_component(PSA_STORAGE_LIB_NAME ${PSA_STORAGE_LIB_FILENAME} NAME [CACHE])
+	set(PSA_LIB_NAME ${PSA_STORAGE_LIB_NAME})
+
+	# The path to the PSA Storage libpsastorage.so (external to this project)
+	# is specified on the cmake command line with the PSA_STORAGE_LD_LIBRARY_PATH
+	# symbol, and used as the the link directory.
+	get_filename_component(PSA_STORAGE_LIB_DIR ${PSA_STORAGE_LIB_FILENAME} DIRECTORY [CACHE])
+	link_directories(${PSA_STORAGE_LIB_DIR})
+
+	# Create list of test binary source files.
+	list(APPEND EXE_SRC ${PSA_ROOT_DIR}/platform/targets/${TARGET}/nspe/common/main.c)
+
+	# Create list of libraries to link to test binary
+	list(APPEND EXE_LIBS
+		${PROJECT_BINARY_DIR}/dev_apis/${_api_dir}/test_combine.a
+		${PROJECT_BINARY_DIR}/val/val_nspe.a
+		${PROJECT_BINARY_DIR}/platform/pal_nspe.a
+		${PROJECT_BINARY_DIR}/dev_apis/${_api_dir}/test_combine.a
```

Comment:
test_combine.a included twice. Please remove this instance.

Reply:
Removed as requested.